.github/workflows/build-test.yml

```diff
@@ -130,7 +128,7 @@ jobs:
             fail-fast: false
             matrix:
                 image: [ 'jdbc-centos7-openjdk8' ]
-                cloud: [ 'AWS' ]
+                cloud: [ 'AWS', 'AZURE', 'GCP' ]
```

Comment:
we don't have to test old driver with AZURE and GCP

Reply:
Done